docker-java-core/src/main/java/com/github/dockerjava/core/command/BuildImageCmdImpl.java

```diff
@@ -355,7 +363,7 @@ public BuildImageCmd withDockerfilePath(String dockerfilePath) {
     @Override
     public BuildImageCmdImpl withTarInputStream(InputStream tarInputStream) {
         checkNotNull(tarInputStream, "tarInputStream is null");
-        this.tarInputStream = tarInputStream;
+        this.tarInputStreams.add(tarInputStream);
```

Comment:
behaviour change: previous implementation was overwriting it, while the new one only adds. Could you please restore the previous one?

Reply:
see comment below.